_posts/2023-01-25-snowflake-snowpro-core-cert.md

```diff
@@ -1,24 +1,26 @@
 ---
-layout: postv2
+layout: post
```

Comment:
What's the difference between these?  Just interested.  Feels like we are going backwards? :grin:

Reply:
Basically styling differences. The second version keeps everything to the centre of the page and adds the heading in a box that looks better. However, the v2 version adds the image that you have inside the `image:` metadata into the top of the blog and it's in a certain size that I didn't like and didn't know how to resize it. So when I added my image tag it was showing the duplicates of the images. That's why I changed it back to older version

Reply:
https://blog.mechanicalrock.io/2023/01/25/snowflake-snowpro-core-cert.html - I only see one image here...?

Will this make the styling inconsistent between blogs though?

Reply:
`I only see one image here` because I removed my image metadata in last PR.
Most people use v1 so it won't be inconsistent